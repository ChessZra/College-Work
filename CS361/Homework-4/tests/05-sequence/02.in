file notafile;/bin/echo hello world;tail tests/01_wc/06.in;file tests/05-sequence/01.env
