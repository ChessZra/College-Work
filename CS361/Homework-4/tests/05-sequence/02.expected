cs361sh> notafile: cannot open `notafile' (No such file or directory)
hello world
wc
wc wc wc
hello test

wc
tests/05-sequence/01.env: empty
cs361sh> 
